dataland-frontend/src/components/pages/UploadEuTaxonomyDataForNonFinancials.vue

```diff
@@ -46,5 +51,11 @@ export default {
       type: String,
     },
   },
-};
+  methods: {
+    /**
+     * Executes a router push to the myDatasets page
+     */
+    redirectToMyDatasets,
```

Comment:
You can also put this in the data block. Then you don't need the additional documentation. But good to know that it also works in the methods block.

Reply:
Changed. Intellij automatically puts it in the methods block on import.

Reply:
Aah. I mean its also fine. But I was just curious